manifests/config.pp

```diff
@@ -2,15 +2,24 @@
 class ntp::config inherits ntp {
 
   if $keys_enable {
-    $directory = ntp_dirname($keys_file)
-    file { $directory:
-      ensure  => directory,
-      owner   => 0,
-      group   => 0,
-      mode    => '0755',
+    file { $keys_file:
```

Comment:
Hmm. Would you consider it backwards-incompatible to start managing this file? Anyone with keys already would get them wiped out when upgrading to this version.